.vscode/launch.json

```diff
@@ -19,6 +19,23 @@
       "port": 4321,
       "restart": true,
       "cwd": "${workspaceRoot}"
+    },
+    {
```

Comment:
Borrowed this from #1790.
It doesn't work for all utests (it depends on the path depth between the test file and package.json) but it works on many of them, and it's only useful if it's in the repo, so I think it's worth merging.